Dockerfile

```diff
@@ -28,6 +28,11 @@ ENTRYPOINT poetry run pytest; poetry run watchmedo shell-command \
     --command='poetry run pytest' \
     .
 
+# -----Configure for local test-----
+FROM test as ci-test
+# Define an entrypoint, and default launch command
+ENTRYPOINT poetry run pytest
+
```

Comment:
Now that you've added a new stage, it would be good to at least briefly document that in your Readme - how does a new user build/run the container for these tests? It's certainly not obvious that the instructions can be parsed from your workflow yml file

Reply:
Fixed as part of Module 9